ReleaseProcessAutomation.IntegrationTests/IntegrationTestSetup.cs

```diff
@@ -38,7 +38,7 @@ public abstract class IntegrationTestSetup : GitBackedTests
   protected TestConsole TestConsole;
 
   [SetUp]
-  public void SetupI ()
+  public virtual void Setup ()
```

Comment:
Class name: base types should be abstract and end in `Base`. Here, it would be IntegrationTestBase